<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProvinsi</class>
 <widget class="QDialog" name="FormProvinsi">
  <property name="windowTitle">
   <string>Form Provinsi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nama Provinsi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputNama"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBatal">
       <property name="text">
        <string>Batal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
